addon/components/kyc-form.js

```diff
@@ -147,6 +148,16 @@ export default Component.extend({
 
   init() {
     this.model = this.get('store').createRecord('identitymind-verification');
+    let user = this.get('user');
+
+    if (user) {
+      this.model.setProperties({
+        bfn: get(user, 'fullLegalName').split(' ')[0],
+        bln: get(user, 'fullLegalName').split(' ')[1] || '',
+        tea: get(user, 'email')
```

Comment:
This might actually be a bit more complex than is needed for this situation. It's a component used in the app, so we could just pass in the values in the template:

```hbs
{{kyc-form defaultFirstName=propA defaultLastName=propB}}
```

Reply:
yeah there are probably a couple better ways to do it, this was the quickest way I could think of before I had to catch a train. Still letting it marinate in my brain.